cupy/cuda/cupy_nccl.h

```diff
@@ -137,7 +139,16 @@ ncclDataType_t _get_proper_datatype(ncclDataType_t datatype) {
     return TYPE2TYPE_V1[datatype];
 }
 
-#endif // #if (NCCL_VERSION < 2000)
+#endif // #if (NCCL_VERSION_CODE < 2000)
+
+#if (NCCL_VERSION_CODE < 2304)
+
+ncclResult_t ncclGetVersion(int *version) {
+    version = 0;
```

Comment:
Is it typo of `version = NCCL_VERSION_CODE;`?

Reply:
No, `NCCL_VERSION_CODE` indicates the version used when compiling CuPy, whereas `ncclGetVersion` returns the version actually used at runtime. I think it's better to report the latter value as `0` rather than `NCCL_VERSION_CODE` to avoid confusion.